server/controllers/conversation.js

```diff
@@ -0,0 +1,75 @@
+const Message = require("../models/Message");
+const Conversation = require("../models/Conversation");
+const mongoose = require("mongoose");
+const asyncHandler = require("express-async-handler");
+
+exports.getAllConversations = asyncHandler(async (req, res, next) => {
```

Comment:
I think some of this code is part of a previous PR? If yes, please update the base of the PR to reflect only new changes